.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: python
 
-python:
-    - "2.7"
+python: "2.7"
 
 sudo: false
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+    directories:
+        - $HOME/.pip-cache/
+        - _build
+
+install: pip install --download-cache $HOME/.pip-cache sphinx==1.1.3
```

Comment:
you could just rely on the default location of the pip cache on Travis: ``$HOME/.cache/pip``: http://docs.travis-ci.com/user/caching/#pip-cache